Shevchuk_Maksym/README.md

```diff
@@ -0,0 +1,93 @@
+Discord Registration and Usage Guide
+Introduction
+Discord is a popular communication platform that enables users to chat, voice call, and video call with individuals and groups. This document provides a step-by-step guide on how to register and use Discord effectively.
+
+Table of Contents
+Creating a Discord Account
+Setting up Discord
+Joining a Server
+Navigating the Discord Interface
+Sending Messages
+Voice and Video Calls
+Managing Notifications
+Customizing Discord
+Privacy and Security
+Troubleshooting
+1. Creating a Discord Account
+To start using Discord, you need to create a user account. Follow these steps:
+
+1)Visit the Discord website at discord.com using your preferred web browser.
+2)Click on the "Login" button located in the top-right corner of the page.
+3)On the login page, click on the "Register" button.
+4)Enter your email address, a secure password, and a unique username.
+5)Complete the reCAPTCHA verification process, if prompted.
+6)Click on the "Continue" button.
+7)Discord may ask you to solve a puzzle to verify that you are not a robot. Follow the instructions provided to complete the verification.
+8)Once verified, you will be directed to the Discord home page, and an email verification link will be sent to the email address you provided.
+9)Check your email and click on the verification link to confirm your account.
+10)Return to the Discord website, and you will be logged in to your new Discord account.
+2. Setting up Discord
+After creating your account, you can customize your Discord profile. Here's how:
```

Comment:
Be consistent. In previous procedure you used "Follow these steps". Use the same phrase for the rest of the procedures.